### ISSUE TITLE
Browse category search has bad ARIA role

### ISSUE THREAD

**Issue by jvine** [sul-dlss/exhibits#1756]
The browse category search input has a role of "browse-search", which is not a valid role.
Example page https://exhibits.stanford.edu/supra/browse/symphonies

`<form class="browse-search-form search-query-form form-horizontal" role="browse-search"  ...`

Replace with `role="search"`

**Comment by jkeck** [comment]
@jvine I'm not seeing this in production currently.  Is this still an issue?

<img width="660" alt="markup screenshot" src="https://user-images.githubusercontent.com/96776/75391375-d2076b00-589e-11ea-957e-980f76e285ac.png">

**Comment by mejackreed** [comment]
Possibly resolved in https://github.com/projectblacklight/spotlight/pull/2480 ?

**Comment by jvine** [comment]
Ack. I took this from a report run a few days ago. Sorry for the duplication!
